deploy.sh

```diff
@@ -0,0 +1,36 @@
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
+then
+    #Clone the latest gh-pages
+    git clone https://github.com/ScottLogic/StockFlux.git --branch gh-pages gh-pages
+
+    #Get line with version from the file -> get the second word -> remove brackets
+    VERSION=$(grep "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    #Get line with the release type (develop/master) from the file -> get the second word -> remove brackets
+    TYPE=$(grep "type" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    rm -rf ./gh-pages/$TYPE
+    cp -r ./public ./gh-pages/$TYPE
+    rm -rf ./gh-pages/$VERSION
```

Comment:
Yeah I see the reasons for it. My only concern is around our version folders accidentally being deleted. It puts the onus on the developer not to execute a silly grunt task to wipe out something old. Whereas without the line if a mistake is made then yes it might be hard to fix, but at least you know all the older stuff is safe? 

Maybe I'm overestimating the "problem" though. Wouldn't mind a third opinion here.